ocs_ci/ocs/resources/pod.py

```diff
@@ -1531,3 +1531,25 @@ def wait_for_pods_to_be_running(
             f"after {timeout} seconds"
         )
         return False
+
+
+def list_of_nodes_running_pods(selector):
+    """
+    The function returns the list of nodes for the given selector
+
+    Args:
+        selector (str): The resource selector to search with
+
+    Returns:
+        list: Pods_running_nodes
```

Comment:
Please make it more informative. like:
"list: a list of nodes that runs the given selector pods"

Reply:
DOne